src/schema/v2/homeView/getSectionsForUser.ts

```diff
@@ -0,0 +1,87 @@
+import { ResolverContext } from "types/graphql"
+
+export async function getSectionsForUser(
+  context: ResolverContext
+): Promise<HomeViewSection[]> {
+  /*
+   * FAKE temporary placeholder logic for determining the sections that a user will see
+   */
+  const { meLoader } = context.authenticatedLoaders
+
+  if (!meLoader) throw new Error("You must be signed in to see this content.")
+
+  const me = await meLoader()
+
+  let sections: HomeViewSection[] = []
+
+  if (me.type === "Admin") {
+    sections = [
```

Comment:
As the comment above loudly notes, this is just some silly placeholder logic for varying the list of sections based on _some_ user attribute. This area of course needs much more definition.